main/src/main/scala/sbt/internal/server/NetworkChannel.scala

```diff
@@ -215,60 +214,43 @@ final class NetworkChannel(
       }
 
     def handleBody(chunk: Vector[Byte]): Unit = {
-      if (isLanguageServerProtocol) {
-        Serialization.deserializeJsonMessage(chunk) match {
-          case Right(req: JsonRpcRequestMessage) =>
-            try {
-              registerRequest(req)
-              onRequestMessage(req)
-            } catch {
-              case LangServerError(code, message) =>
-                log.debug(s"sending error: $code: $message")
-                respondError(code, message, Some(req.id))
-            }
-          case Right(ntf: JsonRpcNotificationMessage) =>
-            try {
-              onNotification(ntf)
-            } catch {
-              case LangServerError(code, message) =>
-                logMessage("error", s"Error $code while handling notification: $message")
-            }
-          case Right(msg) =>
-            log.debug(s"Unhandled message: $msg")
-          case Left(errorDesc) =>
-            logMessage(
-              "error",
-              s"Got invalid chunk from client (${new String(chunk.toArray, "UTF-8")}): $errorDesc"
-            )
-        }
-      } else {
-        contentType match {
-          case SbtX1Protocol =>
-            Serialization
-              .deserializeCommand(chunk)
```

Comment:
The `Serialization` `deserializeCommand` and `serializeCommand` methods are not used anymore. So we should probably remove them.

Reply:
I'll deprecate them for now to avoid bincompat issue.